### PR TITLE
feat: add landing page

### PR DESCRIPTION
**Pull request by anik31**
Landing page for Ecommerce project.
index.html contains the HTML code & style.css contains the CSS code.

Deploy preview - https://deploy-preview-6--vacay-store.netlify.app/

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
UI looks good. Great Work!

File: style.css [hunk 1]
```diff
@@ -135,4 +142,103 @@
...
+.hero-content{
+    position: absolute;
+    width: 100%;
+    height: inherit;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 2rem;
+    /* color: var(--white-color); */
```
Comment (foolhardy21) [inline style.css]: you can remove this comment

File: index.html [hunk 2]
```diff
@@ -4,9 +4,112 @@
...
+                <a href="/product.html" class="category">
+                    <img class="img-round img-round-sizing" src="/assets/t-shirt.jpg" alt="camera">
+                    <text>T-shirts</text>
+                </a>
+                <a href="/product.html" class="category">
+                    <img class="img-round img-round-sizing" src="/assets/mask.jpg" alt="camera">
+                    <text>Mask</text>
+                </a>
+                <a href="/product.html" class="category">
+                    <img class="img-round img-round-sizing" src="/assets/sanitizer.jpg" alt="camera">
```
Comment (foolhardy21) [inline index.html]: the alt attribute has same value

**Review by foolhardy21 — COMMENTED** [review]
Rest is nice. Good to merge after the changes are done.

File: style.css (hunk 1 above)
Comment (anik31) [reply style.css]: done

File: index.html (hunk 2 above)
Comment (anik31) [reply index.html]: did the changes

**Pull request merged**